demo/demo/settings/base.py

```diff
@@ -139,8 +139,8 @@
 REST_FRAMEWORK = {
     'DEFAULT_PAGINATION_CLASS': 'rest_framework.pagination.LimitOffsetPagination',
     'PAGE_SIZE': 100,
-    # Use Django's standard `django.contrib.auth` permissions,
-    # or allow read-only access for unauthenticated users.
+    # Use Django's standard `django.contrib.auth` permissions.
+    # Change to IsAuthenticatedOrReadOnly for read-only unauthenticated access.
```

Comment:
I like this - might be useful to also link to the documentation at https://www.django-rest-framework.org/api-guide/permissions/#setting-the-permission-policy

Reply:
9d30b88f1abdc58419e9ebfda2ff10056623531a